18-temperature-and-humidity-monitor/firmware/src/main.cpp

```diff
@@ -0,0 +1,423 @@
+//
+// Copyright 2022 Blues Inc.  All rights reserved.
+// Use of this source code is governed by licenses granted by the
+// copyright holder including that found in the LICENSE file.
+//
+
+#include <Notecard.h>
+#include <Wire.h>
+#include <SparkFunBME280.h>
+
+// Uncomment this line and replace com.your-company:your-product-name with your
+// ProductUID.
+// #define PRODUCT_UID "com.your-company:your-product-name"
+#define PRODUCT_UID "com.blues.hroche:nf18" 
```

Comment:
you probably want to remove this.

Here's a link to the updated message in the pragma warning.

https://github.com/blues/app-accelerators/blob/main/08-power-monitoring-and-control/firmware/src/app.h#L14

Reply:
Updated the wording but I'd like to keep it as `#error`. I don't see any point in continuing with the build if PRODUCT_UID isn't defined.

Reply:
We make it a warning so the user may choose to configure the productUID as part of the notecard setup, as we've done in other projects.